bot/cogs/booster.py

```diff
@@ -21,15 +28,37 @@ async def on_connect(self) -> None:
     async def _monthly_booster(self) -> None:
         date = datetime.now(pytz.timezone('Asia/Jakarta'))
         if date.day == 1:
-            channel = self.bot.get_channel(ADMIN_CHANNEL_ID)
+            admin_channel = self.bot.get_channel(ADMIN_CHANNEL_ID)
+            tatsu_log_channel = self.bot.get_channel(TATSU_LOG_CHANNEL_ID)
             guild = self.bot.get_guild(GUILD_ID)
             role = guild.get_role(BOOSTER_ROLE_ID)
-            members_content = ''
+            month = date.strftime("%B")
+            member_tag = member_id = ''
+
             for member in role.members:
-                members_content += f"{member.id} "
-            buffer = io.BytesIO(members_content.encode('utf-8'))
-            file = discord.File(buffer, filename="honorary.txt")
-            await channel.send(file=file)
+                await TatsuApi().add_score(member.id, BOOSTER_MONTHLY_EXP)
+                member_tag += f"{member.mention}, "
+                member_id += f"{member.id} "
+                await asyncio.sleep(1.5)
+
+            embed = discord.Embed(
+                title="<a:checklist:1077585402422112297> Score updated!",
+                description=f"Successfully awarded `{BOOSTER_MONTHLY_EXP}` score to <@&{role}> ({len(role.members)} members)\n\n{member_tag}",
```

Comment:
Why don't use `role.mention` to mention the role instead of hardcode it?

Reply:
Oh yeah, I forgot to change it. mb